main.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+import (
+	"github.com/gorilla/mux"
+	"net/http"
+	"github.com/ONSdigital/go-ns/log"
+	"github.com/ONSdigital/go-ns/server"
+	"os"
+	"os/signal"
+	"syscall"
+	"context"
+	"time"
+	"github.com/ONSdigital/dp-auth-api-stub/identity"
+)
+
+func main() {
+	log.HumanReadable = true
```

Comment:
Should this be set from the Makefile?

Reply:
I dont know. I didn't bother with an of the CI stuff because its stub - I assume this is for local testing so you can just run it without all that gumpf